packages/eyes-api/src/input/CheckSettings.ts

```diff
@@ -596,6 +602,15 @@ export class CheckSettingsFluent<TElement = unknown, TSelector = unknown> {
     return this
   }
 
+  lazyLoad(options: LazyLoadOptions): this {
+    this._settings.lazyLoad = {
+      scrollLength: options && options.scrollLength || 300,
+      waitingTime: options && options.waitingTime || 2000,
+      pageHeight: options && options.pageHeight || 15000
```

Comment:
Just a suggestion. You can use modern syntax here since this is a type script and we will compile it anyway.
```suggestion
      scrollLength: options?.scrollLength ?? 300,
      waitingTime: options?.waitingTime ?? 2000,
      pageHeight: options?.pageHeight ?? 15000
```